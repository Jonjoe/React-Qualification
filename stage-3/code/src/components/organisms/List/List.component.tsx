import React from "react";

interface ListProps {}

function List(props: ListProps): JSX.Element {
  return (
    <div>
      <p>List</p>
    </div>
  );
}

export default List as React.FC<ListProps>;
