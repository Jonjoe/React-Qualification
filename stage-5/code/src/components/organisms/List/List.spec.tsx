import React from "react";
import renderer from "react-test-renderer";

import List from "./List.component";

test("Link changes the class when hovered", () => {
  const component = renderer.create(
    <List data={["this", "is", "a", "list"]} />
  );

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
