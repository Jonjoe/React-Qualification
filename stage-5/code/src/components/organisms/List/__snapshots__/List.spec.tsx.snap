// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<ul
  className="sc-AxjAm juAXlR"
>
  <li>
    <a />
  </li>
  <li>
    <a />
  </li>
  <li>
    <a />
  </li>
  <li>
    <a />
  </li>
</ul>
`;
